resources/js/features/home/components/+root/FrontPageNews/NewsCard.tsx

```diff
@@ -90,6 +91,8 @@ const NewsCardImage: FC<NewsCardImageProps> = ({ src }) => {
 
       {isImageValid ? (
         <div
+          role="img"
+          aria-label="news post photo"
           className="h-28 w-full rounded bg-cover bg-center sm:w-[197px]"
```

Comment:
Google is angry because this element doesn't have a role or aria-label. It's fine that this is a plain English string.